challenges/functions.js

```diff
@@ -6,6 +6,13 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
+// const names = ["Jerry", "Nerma"];
+// const ages = ["27", "23"];
```

Comment:
leaving your old code in with comments is good for reference of the logic you took to get to the point of properly working code